spark-extension/src/main/scala/org/apache/spark/sql/auron/common/AuronSQLConf.scala

```diff
@@ -0,0 +1,168 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.auron.common
+
+import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.sql.internal.SQLConf.buildConf
+
+object AuronSQLConf extends SQLConf {
+
+  val UDF_UDF_JSON_ENABLED =
+    buildConf("spark.auron.udf.UDFJson.enabled")
+      .doc("Enable native implemented get_json_object/json_tuple. May introduce inconsistency in special case (especially with illegal json inputs).")
+      .booleanConf
+      .createWithDefault(true)
+
+  val UDF_BRICK_HOUSE_ENABLED =
+    buildConf("spark.auron.udf.brickhouse.enabled")
+      .doc("Enable some native-implemented brickhouse UDFs.")
+      .booleanConf
+      .createWithDefault(true)
+
+  val DECIMAL_ARITH_OP_ENABLED =
+    buildConf("spark.auron.decimal.arithOp.enabled")
+      .booleanConf
+      .createWithDefault(false)
+
+  val ENABLED_SCAN =
+    buildConf("spark.auron.enable.scan")
```

Comment:
Could you fix the naming style of these configuration keys? For example, change `spark.auron.enable.scan` to `spark.auron.scan.enabled`